From: Katharina Drexel <katharina.drexel@bfh.ch>
Date: Fri, 13 May 2022 13:32:06 +0200
Subject: Add tests to group functional

Not all tests are working as they must run in a composer environment.

diff --git a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/FragmentTest.php b/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/FragmentTest.php
index b514ed3b8e..27d869cb20 100644
--- a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/FragmentTest.php
+++ b/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/FragmentTest.php
@@ -15,6 +15,7 @@ class FragmentTest extends AbstractWebTestCase
 {
     /**
      * @dataProvider getConfigs
+       @group functional
      */
     public function testFragment($insulate)
     {
diff --git a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ProfilerTest.php b/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ProfilerTest.php
index 7b65ca5c27..839b581a45 100644
--- a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ProfilerTest.php
+++ b/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ProfilerTest.php
@@ -15,6 +15,7 @@ class ProfilerTest extends AbstractWebTestCase
 {
     /**
      * @dataProvider getConfigs
+     * @group functional
      */
     public function testProfilerIsDisabled($insulate)
     {
@@ -38,6 +39,7 @@ class ProfilerTest extends AbstractWebTestCase
 
     /**
      * @dataProvider getConfigs
+     * @group functional
      */
     public function testProfilerCollectParameter($insulate)
     {
diff --git a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SessionTest.php b/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SessionTest.php
index ae8c7afafd..069f4aa674 100644
--- a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SessionTest.php
+++ b/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/SessionTest.php
@@ -17,6 +17,7 @@ class SessionTest extends AbstractWebTestCase
      * Tests session attributes persist.
      *
      * @dataProvider getConfigs
+     * @group functional
      */
     public function testWelcome($config, $insulate)
     {
@@ -50,6 +51,7 @@ class SessionTest extends AbstractWebTestCase
      * Tests flash messages work in practice.
      *
      * @dataProvider getConfigs
+     * @group functional
      */
     public function testFlash($config, $insulate)
     {
@@ -74,6 +76,7 @@ class SessionTest extends AbstractWebTestCase
      * polluting each other's session data.
      *
      * @dataProvider getConfigs
+     * @group functional
      */
     public function testTwoClients($config, $insulate)
     {
@@ -130,6 +133,7 @@ class SessionTest extends AbstractWebTestCase
 
     /**
      * @dataProvider getConfigs
+     * @group functional
      */
     public function testCorrectCacheControlHeadersForCacheableAction($config, $insulate)
     {
